packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBar.tsx

```diff
@@ -65,14 +65,14 @@ const getStyles = createStyle(() => {
     `,
     logMarker: css`
       label: logMarker;
-      background-color: rgba(0, 0, 0, 0.5);
+      background-color: ${theme.colors.text};
```

Comment:
This looks wrong , use bg1 or bg2 or bg3 for backgrounds

Reply:
or are these text-like elements?

Reply:
not really. I just needed a color that is the reverse of bgcolor. so dark in light theme and light in dark theme.

Reply:
can you show what elements in the UI is the logMarker? maybe border2 can work as an alternative

Reply:
These are the log markers
![Screenshot 2021-02-23 at 16 21 59](https://user-images.githubusercontent.com/13729989/108865345-74157c80-75f3-11eb-8652-e10342c43453.png)

border2 would be light for light which wouldn't work.